transactions.md

```diff
@@ -6,9 +6,61 @@ One transaction event occurs at authorisation time. The other transaction event
 
 ## Transaction Object
 
-##### API version from `2022-07-13`
+| **Field name** | **Description** | **Type** |
+| --- | ------ | --- |
+| `accountId` | The unique identifier of the user account at Fidel API | string |
+| `amount` | The amount of the transaction in the currency it was charged in | number |
+| `approvalCode` | Unique code that is sent along with the authorization of the transaction by Amex. It mirrors the identifiers.amexApprovalCode property. | string (or null) |
+| `auth` | Indicates whether Fidel API received an authorization event for this transaction | boolean |
+| `authCode` | Unique code that is sent along with the authorization of the transaction by Visa and Mastercard. It is sent by the issuer/the PCN on the issuer's behalf when approving a transaction. It mirrors the identifiers.mastercardAuthCode or identifiers.visaAuthCode properties, depending on the issuing card for the transaction. | string (or null) |
+| `brand.id` | The identifier of the brand at Fidel API | string |
+| `brand.logoURL` | The URL of the logo for the brand | string (or null) |
+| `brand.name` | The name of the merchant where the purchase was made | string |
+| `brand.metadata` | Customized information you added to the brand | (can be null) |
+| `card.firstNumbers` | The first numbers of the card, used for helping users identify their cards | string |
+| `card.id` | The tokenized card identifier created by Fidel API | string |
+| `card.lastNumbers` | The last numbers of the card, used for helping users identify their cards | string |
+| `card.metadata` | Customized information you added to the card | (can be null) |
```

Comment:
Missing object type?

Reply:
Added object type for consistency and "(or null)" as in the others.